### ISSUE TITLE
Feature: Tydelig vise forskjell på definisjoner som er ment for ulike målgrupper i begrepskatalogen i portal

### ISSUE THREAD

**Issue by KjerstiSteien** [Informasjonsforvaltning/fdk-issue-tracker#972]
### Feature beskrivelse
Vise tydelig forskjell mellom definisjoner for ulike målgrupper (uten målgruppe, for allmennheten, for spesialister. 

### Hvorfor trenger vi det?
For at det skal være enkelt for brukere av katalogen å finne frem til definisjonen de har behov for, og enkelt gjenkjenne hvilken målgruppe definisjonene er ment som. 

Slik ser det ut i dag: 
![image](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/assets/110605570/7ff569ca-bb21-4871-bc65-33c16c64efc2)

Merk at teksten som vises for de ulike definisjonene er selve definisjonen i dette tilfellet. Et reelt eksempel ville vært at det ikke fremkom av teksten hvilken målgruppe definisjonen var ment som.

**Comment by KjerstiSteien** [comment]
@Lillebo kan du se på design på denne? Kan godt tas når vi kommer til nytt design for detaljside for begrep.

**Comment by Lillebo** [comment]
Ja, tenker også at det hadde vært fint å se på dette samlet når vi går løs på detaljsiden som helhet 👍
